packages/webpack/src/config/utils/postcss.js

```diff
@@ -129,6 +129,11 @@ export default class PostcssConfig {
         config = merge({}, this.defaultConfig, config)
         this.loadPlugins(config)
       }
+      if (config.options && config.options.pushCssNano) {
```

Comment:
This option would be better to move under `build` with default value inside `config` package :)

Reply:
Where would you put t exactly? I thought using an options object inside postcss is a good idea (`build.postcss.options`) 🤔

Reply:
oh my bad! this `config.options` is already scoped 👍